src/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -647,14 +651,77 @@ public boolean onOptionsItemSelected(MenuItem item) {
                     onBackPressed();
                 }
                 break;
-
+            case R.id.action_sort_descending:
+                item.setChecked(!item.isChecked());
+                boolean isAscending = !item.isChecked();
+                com.owncloud.android.db.PreferenceManager.setSortAscending(isAscending,this,FileStorageUtils.FILE_DISPLAY_SORT);
```

Comment:
Think `com.owncloud.android.db` is not needed, you can just call PreferenceManager directly since it was properly imported at the top of the class. Check the whole class since you are using it in more places.